Test_Engine/Create/FailResult.cs

```diff
@@ -41,9 +42,9 @@ public static partial class Create
 
         public static TestResult FailResult(string errorMessage, string description = "", bool isCritical = false)
         {
-            ResultStatus status = isCritical ? ResultStatus.CriticalFail : ResultStatus.Fail;
-            List<Event> events = new List<Event> { new Event { Type = EventType.Error, Message = errorMessage } };
-            return new TestResult(status, events, description);
+            TestStatus status = isCritical ? TestStatus.Error : TestStatus.Warning;
+            //List<Event> events = new List<Event> { new Event { Type = EventType.Error, Message = errorMessage } };
```

Comment:
Think it might be. This is part of code that @adecler will update in an upcoming PR. I just tweaked it to compile, and while doing so, tried to keep things I could not properly update commented out. Agree that we should never leave commented out code in the general case, but I think this should be ok as an exception.

Reply:
It can be safely deleted 👍